app/scripts/lib/auth-errors.js

```diff
@@ -225,6 +225,27 @@ function (_, Errors) {
     SIGNUP_DISABLED_BY_RELIER: {
       errno: 1030,
       message: t('Signup has been disabled')
+    },
+    ALL_NUMBERS_LETTERS: {
+      errno: 1031,
+      message: 'ALL_NUMBERS_LETTERS'
```

Comment:
Can you just change all of these `message`s to `UNEXPECTED_ERROR_MESSAGE`? The message is not yet presented to the user.